### PR TITLE
fix entity inspector editor

### PR DESCRIPTION
**Pull request by rushan-zagidullin**
There were two bugs in entityInspector.edit screen.
1. Entity Foo has ManyToMany relation with EntitySnapshot, that has getLabel metaProperty. When you try to open entity Foo over entity inspector you will catch exception.
2. Entity Foo has readOnly MetaProperty "myProperty" related to other non local properties.
During view creation, "myProperty" will be scipped, but component for that property will be created and again you will get UnfetchedAttributeException.

### CODE REVIEW

File: modules/global/src/com/haulmont/cuba/core/global/MetadataTools.java [hunk 1]
```diff
@@ -706,6 +706,22 @@ public String getDatabaseColumn(MetaProperty metaProperty) {
     }
 
     /**
+     * Determine whether the given metaProperty relates to at least one non local property
+     */
+    public boolean isRelatedToNonLocalProperty(MetaProperty metaProperty) {
```
Comment (knstvk) [inline modules/global/src/com/haulmont/cuba/core/global/MetadataTools.java]: Move it to EntityInspectorEditor, as the method looks trivial and rarely needed.

**Comment by knstvk** [comment]
Please provide a test project with the data model where the issue is reproduced.

**Comment by rushan-zagidullin** [comment]
Ok, I will make demo project

**Comment by rushan-zagidullin** [comment]
Demo project https://github.com/rushan-zagidullin/cubaEntityInspectorEditorIssueSample/tree/master

**Comment by knstvk** [comment]
Thank you!

**Pull request merged**